__manifest__.py

```diff
@@ -20,13 +20,24 @@
     'version': '0.1',
 
     # any module necessary for this one to work correctly
-    'depends': ['base'],
+    'depends': ['base','sale_management','product','stock','web','website_sale','base_geolocalize',],
```

Comment:
dependencies are installed in a hierarchical manner, so you would notice we don't need to mention some of the modules as they are getting installed as dependencies of other modules (please refer to their manifests)
```suggestion
    'depends': ['sale_management','stock','website_sale','base_geolocalize',],
```